src/main/java/com/coveros/selenified/element/check/Matches.java

```diff
@@ -36,14 +35,9 @@
  * @version 3.1.0
  * @lastupdate 2/21/2019
  */
-public class Matches extends Assert {
+public interface Matches extends Check {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1214)